### PR TITLE
Slightly buffs plasmemes

### PR DESCRIPTION
**Pull request by Tacolizard**
:cl: #PlasmemeLivesMatter
tweak: plasmamen no longer take double burn and heat damage
/:cl:

### CODE REVIEW

**Comment by PKPenguin321** [comment]
Nah

**Comment by Shadowlight213** [comment]
Nah

**Comment by optimumtact** [comment]
Low effort pr again, with poor implementation on a bad idea.

Please try to listen to the last piece of advice I gave you.

File: code/modules/mob/living/carbon/human/species_types.dm [hunk 1]
```diff
@@ -738,8 +738,8 @@ var/global/image/plasmaman_on_fire = image("icon"='icons/mob/OnFire.dmi', "icon_
 	mutantlungs = /obj/item/organ/lungs/plasmaman
 	dangerous_existence = 1 //So so much
 	blacklisted = 1 //See above
-	burnmod = 2
-	heatmod = 2
+	//burnmod = 2
```
Comment (KorPhaeron) [inline code/modules/mob/living/carbon/human/species_types.dm]: lmao

**Pull request closed without merging**